modules/cpmso-openstack-ts-az-config.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+//
+// * machine_management/control_plane_machine_management/cpmso-troubleshooting.adoc
+
+:_content-type: REFERENCE
+[id="cpmso-openstack-ts-az-config_{context}"]
+= Configuring {rh-openstack} Availability Zones type
```

Comment:
Yeah, @maxwelldb any better suggestion? 
"Configuring {rh-openstack} clusters for availability zones"? Since we are configuring the clusters with respect to AZs so that the control plane machine is created.

Reply:
Configuring upgraded clusters that run on {rh-openstack} and use availability zones?

Reply:
Or "Configuring {rh-openstack} with compute availability zones" [OpenShift on OpenStack with compute Availability Zones: Missing rootVolume availability zone](https://access.redhat.com/solutions/7024383)

and for the other one "Configuring {rh-openstack} with availability zones". [penShift on OpenStack with Availability Zones: Invalid Compute ServerGroup setup during OpenShift deployment](https://access.redhat.com/solutions/7013893)

How about this?

Reply:
@jeana-redhat Any suggestions???